client/common/src/blockchain_utils.rs

```diff
@@ -48,3 +50,28 @@ pub fn get_events_at_block(
         .transpose()?
         .ok_or(EventsRetrievalError::StorageNotFound)
 }
+
+/// Attempt to convert BoundedVec of BoundedVecs of bytes.
+///
+/// Returns a list of [`Multiaddr`] objects that have successfully been parsed from the raw bytes.
+pub fn convert_raw_multiaddresses_to_multiaddr(multiaddresses: Multiaddresses) -> Vec<Multiaddr> {
```

Comment:
Not sure this fits in `blockchain_utils`, but I don't mind much anyway.

Reply:
We were using this in the blockchain service and in the task, so I figured this is a common area to put it in.